db/sql/V1.1__tracker_configuration.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE ticket_tracker_servers (
+    id UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+    name TEXT NOT NULL,
+    url TEXT NOT NULL,
```

Comment:
Yes, because different services user could have different access to projects inside the same server URL. Maybe the name of the table is a little misleading. We can discuss this in a quick VC.

Reply:
Okey makes sense. I didn't have in mind that use case, thanks for the clarification.